react-component-template/index.jsx

```diff
@@ -12,19 +14,8 @@ export default class {{PascalName}} extends Component {
     name: '{{titleName}}'
   }
 
-  constructor (props) {
-    super(props)
-    // initialize state values
-    this.state = {}
-  }
-
-  // use the pure-render mixin without the mixin. This allows us to use es6
-  // classes and avoid "magic" code. NOTE: if this component is used directly
-  // by react-router, you should delete it, otherwise, the <Link> component will
-  // not cause a re-render
-  shouldComponentUpdate (...args) {
-    return shouldComponentUpdate.apply(this, args)
-  }
+  // initialize state values
+  state = {}
```

Comment:
My thinking is, since some things will be mutating initial `state` via `props` (ie - require a constructor), and some won't have state at all (ie - not requiring a `state` variable), the number of components that will need a state, but aren't mutating props, will be small. So, let's just set up our initial state/constructor each time, depending on what we need.